internal/controller/expected_netboxmock_calls_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2024.
```

Comment:
This has been fixed in https://github.com/netbox-community/netbox-operator/pull/3!